src/test/java/org/folio/rest/impl/SendingEmailTest.java

```diff
@@ -284,4 +297,233 @@ private static void assertSentMessageContainsHeaders(WiserMessage message,
     assertEquals(expectedHeaders, actualHeaders);
   }
 
+  @Test
+  public void shouldSucceedWhenBothLocalAndRemoteConfigsExistAndAreValid() throws Exception {
+    int mockServerPort = userMockServer.port();
```

Comment:
do we have to initialize it it every single test? can we store it in a field?

Reply:
This is how it was done in all other tests. Anyway, field extracted.